pkg/assembler/backends/neo4j/certifyScorecard.go

```diff
@@ -203,3 +204,135 @@ func setCertifyScorecardValues(sb *strings.Builder, certifyScorecardSpec *model.
 		queryValues["collector"] = certifyScorecardSpec.Collector
 	}
 }
+
+func (c *neo4jClient) CertifyScorecard(ctx context.Context, source model.SourceInputSpec, scorecard model.ScorecardInputSpec) (*model.CertifyScorecard, error) {
+	session := c.driver.NewSession(neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
+	defer session.Close()
+
+	values := map[string]any{}
+	values["sourceType"] = source.Type
+	values["namespace"] = source.Namespace
+	values["name"] = source.Name
+
+	if source.Commit != nil && source.Tag != nil {
+		if *source.Commit != "" && *source.Tag != "" {
+			return nil, gqlerror.Errorf("Passing both commit and tag selectors is an error")
+		}
+	}
+
+	if source.Commit != nil {
+		values["commit"] = *source.Commit
+	}
+
+	if source.Tag != nil {
+		values["tag"] = *source.Tag
+	}
+
+	values["timeScanned"] = scorecard.TimeScanned
```

Comment:
can use the constant defied above for these.

Reply:
Changed to use constants